src/modelfree/tsne/visualize.py

```diff
@@ -41,8 +43,17 @@ def main_config():
 @visualize_ex.named_config
 def inline_config():
     styles = ['paper', 'twocol']
-    pretty_labels = False  # make legend smaller
+    internal_legend = True
     external_legend_params = None
+    pretty_labels = {
+        'Adv': 'Adv',
+        'Zoo': 'Zoo',
+        'Rand': 'Rand',
+    }
```

Comment:
This seems like it's not doing anything right now? Is the argument for keeping it around that we want to keep the pattern of doing a label lookup even though it's currently a no-op?

Reply:
It's doing something in the sense that `_make_handles` needs a `pretty_labels` argument. In the default config, `pretty_labels` is not the identity mapping. Here it happens to be the identity, because the keys are already pretty short, and we want a short label for inline legends.

I guess I could replace `pretty_labels` here with some dict-like class that is just an identity, but I'm not sure if this would be shorter or clearer.